src/simplay-web/src/EventFactory.ts

```diff
@@ -0,0 +1,147 @@
+import { ContainerSetCapacityEvent } from './ContainerSetCapacityEvent';
+import { ContainerSetCapacityEventArgs } from './ContainerSetCapacityEventArgs';
+import { ContainerSetLevelEvent } from './ContainerSetLevelEvent';
+import { ContainerSetLevelEventArgs } from './ContainerSetLevelEventArgs';
+import { EventAction } from './EventAction';
+import { EventArgs } from './EventArgs';
+import { EventArgsCtor } from './EventArgsCtor';
+import { EventCtor } from './EventCtor';
+import { EventSerialized } from './EventSerialized';
+import { MoveNearCellEvent } from './MoveNearCellEvent';
+import { MoveNearCellEventArgs } from './MoveNearCellEventArgs';
+import { MoveNearEvent } from './MoveNearEvent';
+import { MoveNearEventArgs } from './MoveNearEventArgs';
+import { ResourceSetCapacityEvent } from './ResourceSetCapacityEvent';
+import { ResourceSetCapacityEventArgs } from './ResourceSetCapacityEventArgs';
+import { ResourceSetUtilizationEvent } from './ResourceSetUtilizationEvent';
+import { ResourceSetUtilizationEventArgs } from './ResourceSetUtilizationEventArgs';
+import { SetDecoratingTextEvent } from './SetDecoratingTextEvent';
+import { SetDecoratingTextEventArgs } from './SetDecoratingTextEventArgs';
+import { SetInteractingEvent } from './SetInteractingEvent';
+import { SetInteractingEventArgs } from './SetInteractingEventArgs';
+import { SetNotInteractingEvent } from './SetNotInteractingEvent';
+import { SetNotInteractingEventArgs } from './SetNotInteractingEventArgs';
+import { SetPositionEvent } from './SetPositionEvent';
+import { SetPositionEventArgs } from './SetPositionEventArgs';
+import { SetSpriteFrameEvent } from './SetSpriteFrameEvent';
+import { SetSpriteFrameEventArgs } from './SetSpriteFrameEventArgs';
+import { SetTintColorEvent } from './SetTintColorEvent';
+import { SetTintColorEventArgs } from './SetTintColorEventArgs';
+import { SetVisibleEvent } from './SetVisibleEvent';
+import { SetVisibleEventArgs } from './SetVisibleEventArgs';
+import { StoreSetCapacityEvent } from './StoreSetCapacityEvent';
+import { StoreSetCapacityEventArgs } from './StoreSetCapacityEventArgs';
+import { StoreSetContentEvent } from './StoreSetContentEvent';
+import { StoreSetContentEventArgs } from './StoreSetContentEventArgs';
+import { Event } from './Event';
+
+export class EventFactory {
```

Comment:
This class only has static members, what is the advantage of making it a class?
Could we not just provide a function as we do in `Grid.ts`